clang/lib/CodeGen/BackendUtil.cpp

```diff
@@ -1162,6 +1165,7 @@ void EmitAssemblyHelper::RunCodegenPipeline(
   // does not work with the codegen pipeline.
   // FIXME: make the new PM work with the codegen pipeline.
   legacy::PassManager CodeGenPasses;
+  std::unique_ptr<MachineModuleInfo> MMI;
```

Comment:
can this just be `MachineModuleInfo MMI;` below?

Reply:
I'll change it to a normal construction with the `TM`.